test/parallel/test-buffer-indexof.js

```diff
@@ -81,43 +81,54 @@ assert.equal(Buffer.from('ff').indexOf(Buffer.from('f'), 1, 'ucs2'), -1);
 // test hex encoding
 assert.equal(
     Buffer.from(b.toString('hex'), 'hex')
-    .indexOf('64', 0, 'hex'), 3);
+    .indexOf('64', 0, 'hex'),
```

Comment:
And, in case you're curious, that rule finds 110 instances of mis-aligned chained properties in 18 files (3 in lib and 15 in test).